docs/reference/builder.md

```diff
@@ -608,7 +608,7 @@ FROM [--platform=<platform>] <image>[@<digest>] [AS <name>]
 ```
 
 The `FROM` instruction initializes a new build stage and sets the
-[*Base Image*](../../glossary/#base-image) for subsequent instructions. As such, a
+[*Base Image*](https://docs.docker.com/glossary/#base_image) for subsequent instructions. As such, a
```

Comment:
I guess this was originally linking to https://github.com/docker/cli/blob/master/docs/reference/glossary.md#base-image, but that page is not published on docs.docker.com. Perhaps we should remove it here